generic/tests/cfg/pxe_boot.cfg

```diff
@@ -9,17 +9,20 @@
     image_size_pxe = 1G
     force_create_image_pxe = yes
     remove_image_pxe = yes
-    boot_once = n
     kill_vm_on_error = yes
     network = bridge
     restart_vm = yes
     pxe_timeout = 360
     image_verify_bootable = no
     kill_vm = yes
     kill_vm_gracefully = no
+    del boot_order
+    del boot_once
+    del boot_reboot_timeout
+    del boot_splash_time
+    bootindex_nic1 = 0
```

Comment:
Although it is the default setting in avocado-vt,  How about add `nics = nic1` directly in test case cfg ?

Reply:
that's okay.